cmd/server/assets/admin/users/show.html

```diff
@@ -47,24 +47,30 @@ <h6 class="mb-2">System admin</h6>
       </div>
     </div>
 
-    <div class="card mb-3 shadow-sm">
-      <div class="card">
+    {{if $user.Realms}}
```

Comment:
Alternatively leave the card and
```
{{else}
This user is not a member of any realms
```

might make the UI more consistent / clear

Reply:
Done